client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx

```diff
@@ -40,7 +40,9 @@ export const MostPopular: React.FC< MostPopularProps > = ( {
 
 	return (
 		<div className={ wrapperClassName }>
-			<h2 className="jetpack-product-store__most-popular--heading">{ heading }</h2>
+			{ heading ? (
+				<h2 className="jetpack-product-store__most-popular--heading">{ heading }</h2>
+			) : null }
```

Comment:
```suggestion
			{ heading && (
				<h2 className="jetpack-product-store__most-popular--heading">{ heading }</h2>
			) }
```

Nitpick, I think this would do the same thing and is slightly cleaner 😄

Reply:
Of course!